### ISSUE TITLE
Training scripts

### ISSUE THREAD

**Issue by wyddmw** [Sharpiless/Train-RAFT-from-single-view-images#1]
Hi, thanks for the code release of this great work. I wanna ask is the provided training script same as the official training settings in the paper?

**Comment by Sharpiless** [comment]
Not exactly. The code synthesizes the data online, resulting in more varied but erratic training. We train the model under the same camera setup on COCO dataset with this code and get slightly worse EPE (3.58 vs 3.44) but better Fl (9.88 vs 10.44).

**Comment by wyddmw** [comment]
Thanks for your in-time reply!  Except for the implementation of dataloader, I find the hyper-parameters a little different from the setting mentioned in the paper. I'd like to reimplement the training on the CoCo dataset and could you please provide the training scripts or the related settings used for the official training in the paper?

**Comment by Sharpiless** [comment]
Thank you for your attention! I'll update the Settings in the code in a few days.
